mne/viz/evoked.py

```diff
@@ -311,10 +311,30 @@ def _plot_evoked(evoked, picks, exclude, unit, show,
                         for i in bad_ch_idx:
                             if i in idx:
                                 colors[idx.index(i)] = 'r'
-                    for ch_idx in range(len(D)):
+
+                    # find the channels with the least activity
+                    # to map them in front of the more active ones
+                    if zorder == 'std':
+                        def zorder_(D):
+                            return D.std(axis=1).argsort()
+                    elif zorder == 'unsorted':
+                        def zorder_(D):
+                            return list(range(D.shape[0]))
+                    elif not callable(zorder):
+                        error = ('`zorder` must be a function, "std" '
+                                 'or "unsorted", not {}.')
+                        raise TypeError(error.format(type(zorder)))
+                    else:
+                        zorder_ = zorder
+
+                    z_ord = zorder_(D)
+
+                    # plot channels
+                    for ch_idx, z in enumerate(z_ord):
                         line_list.append(ax.plot(times, D[ch_idx], picker=3.,
-                                                 zorder=1,
+                                                 zorder=z,
                                                  color=colors[ch_idx])[0])
+
                 if gfp:  # 'only' or boolean True
                     gfp_color = 3 * (0.,) if spatial_colors else (0., 1., 0.)
```

Comment:
Ah, fixing it, I see why it's like this: it's > 79 chars if you do it the long and readable way. Will leave.